test/engines/wintermute/path_utils.h

```diff
@@ -0,0 +1,229 @@
+#include <cxxtest/TestSuite.h>
+#include "engines/wintermute/utils/path_util.h"
+/**
+ * Test suite for the functions in engines/wintermute/utils/path_util.h
+ *
+ * NOTE: This is not a prescription;
+ *       this was not written by the original engine author;
+ *       this was not written by the engine porter.
+ *
+ * It might, however, help to spot variations in behavior that are introduced by modifications
+ */
+
+class PathUtilTestSuite : public CxxTest::TestSuite {
+	public:
+	const Common::String unixPath;
+	const Common::String unixCapPath;
+	const Common::String windowsPath;
+	const Common::String windowsCapPath;
+	const Common::String emptyString;
+	const Common::String dualExtPath;
+	const Common::String manyExtPath;
+	const Common::String mixedSlashesPath1;
+	const Common::String mixedSlashesPath2;
+	const Common::String unixRelativePath;
+	const Common::String windowsRelativePath;
+	PathUtilTestSuite () :
+		unixPath("/some/file.ext"),
+		unixCapPath("/SOME/FILE.EXT"),
```

Comment:
I am an idiot, my commit just reinstates the previous behaviour. Sorry, it's been a few months.